woocommerce/woocommerce-ios
# [UL&S] Better error message when entering a non-valid url address

Closes #3325 

Before:
<img src="https://user-images.githubusercontent.com/2722505/102297353-137e1980-3f8a-11eb-8678-68161159b640.png" width="350"/>

After:
<img src="https://user-images.githubusercontent.com/2722505/102297457-49230280-3f8a-11eb-9df2-86fca308b4c4.gif" width="350"/>

The original error was fired by WPAuthenticator. This PR builds upon the mechanism implemented to take over errors emitted by WPAuthenticator: wordpress-mobile/WordPressAuthenticator-iOS#525

## Changes
* Add a new case to the Authenticator error (the domain error we use in the host app to map errors thrown by WPAuthenticator)
* Present the same error we present when we don't find a valid WP site (as discussed in the [issue comments](https://github.com/woocommerce/woocommerce-ios/issues/3325#issuecomment-744287061))

## How to test
* Check out the branch. Run `bundle exec pod install`
* Attempt to login with a Store Address, enter a not-valid site (i.e. GitHub.cox)

Update release notes:
